src/libchessdetector/chess_detector.cpp

```diff
@@ -32,7 +32,10 @@ ChessDetector::~ChessDetector()
 		m_resp = NULL;
 		m_resp_init = false;
 	}
-
+	if(m_points)
```

Comment:
you can remove the `if(m_points)`

Reply:
Should be fixed here : dbb5a9ad8227d426c79a983b36ea7fec86b957b0